themes/powerlevel10k_classic.omp.json

```diff
@@ -26,7 +26,7 @@
           "background": "#546E7A",
           "properties": {
             "style": "short",
-            "postfix": " "
+            "postfix": "<#546E7A>\u2588</>"
```

Comment:
What if we swapped the default value of `prefix` to this character? This isn't the only theme having this issue on startup on Powershell, and I like this crazy out-of-the-box idea.